pkg/sql/logictest/testdata/logic_test/unique

```diff
@@ -192,7 +192,10 @@ INSERT INTO uniq_fk_child VALUES (1, 1, 1), (2, 2, 2)
 
 # This fails the foreign key checks but passes the uniqueness checks.
 statement error pgcode 23503 pq: insert on table "uniq_fk_child" violates foreign key constraint "fk_b_ref_uniq_fk_parent"\nDETAIL: Key \(b, c\)=\(3, 3\) is not present in table "uniq_fk_parent"\.
-INSERT INTO uniq_fk_child VALUES (3, 3, 3), (4, 4, 4)
+INSERT INTO uniq_fk_child VALUES (3, 3, 3)
```

Comment:
you could potentially have both as well, and just check `pq: insert on table "uniq_fk_child" violates foreign key constraint "fk_b_ref_uniq_fk_parent"` -- otherwise LGTM